### DISCUSSION TITLE
Aureal Vortex sound cards emulation

### DISCUSSION THREAD

**Discussion by Torinde** [86Box/86Box#2911] [Ideas]
It would be nice to see the emulation of Aureal Vortex sound cards, but the documentation for those cards is probably lost media :(
I am also curious if the 86box will (in the distant future) emulate Pentium III, IV and newer CPUs together with motherboards, will it emulate AC'97 (and maybe even HD Audio) sound cards?

_Originally posted by @MrPepka in https://github.com/86Box/86Box/discussions/2402#discussioncomment-3019121_

Link to a site with a lot of info on Aureal3D joncampbell123/dosbox-x/issues/3597

**Comment by OBattler** [comment]
Converting to discussion per the policy.

**Comment by MrPepka** [comment]
Yes, there is certainly a lot of information about Aureal3D, I even found an SDK for it once, but it's not enough. There is a need for documentation of these cards describing, above all, registers, principles of card operation, etc

**Comment by Torinde** [comment]
[Aureal Linux Hacking](https://aureal.sourceforge.net) - usable drivers for Linux, "The current reverse engineering work can be checked out of the cvs using the "aureal_re" module"

**Reply by Torinde** [reply]
[A3D-to-Vista+ wrapper](https://www.pcgamingwiki.com/wiki/Glossary:Sound_card#A3D)

**Reply by Cacodemon345** [reply]
One never reverse-engineered the A3D support, the other one is exclusively an user-space wrapper.

**Reply by Torinde** [reply]
open source repositories with sound card drivers - covering Aureal models among others: [OSS project](http://www.opensound.com/oss.html), [FreeBSD sound driver sources](https://cgit.freebsd.org/src/tree/sys/dev/sound/pci/hda), [ALSA drivers](https://alsa-project.org/), [IBM Audio Interface Library for 32-bit DOS (AIL/32) v.1.05](https://github.com/Wohlstand/ail32-sandbox)

**Comment by jadasse** [comment]
86Box has no plans to emulate Pentium III mobos and later. Also, AC'97 is already emulated.

**Comment by Torinde** [comment]
[Games With True 3D Spatial Audio (Ambisonics, Binaural, HRTF, Dolby Atmos for Headphones) [Non-VR & VR]](https://www.reddit.com/r/GameAudio/comments/b428zx/games_with_true_3d_spatial_audio_ambisonics/) - Sensuara employee commenting on Reddit - maybe can be contacted to help with questions about the specs, emulation, etc.

**Reply by Cacodemon345** [reply]
I bet he's tight-lipped by contract.

**Reply by Torinde** [reply]
Worth to try, if somebody can, maybe he's not anymore?

Related: [ABIT UA11 - Sensaura driver](http://www.worknd.ru/abit_ua11.html): DirectSound3D, Sensaura ZoomFX, I3DL2, Creative EAX 1.0, 2.0 and 3.0
